File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> numberSet = new HashSet<>(); 
        for(int num : nums){ 
            numberSet.add(num); 
        }

        int longestSeq = 0; 

        for (int num : numberSet){ 
            if(!numberSet.contains(num -1)){ 
                int length = 1; 
                int currNum = num; 
                while(numberSet.contains(currNum + 1)){
                    currNum++; 
                    length++; 
                }
                if(length > longestSeq){
                    longestSeq = length; 
                }
            }
        }


        return longestSeq;

    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-4.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dicti = {} 
        for i, num in enumarate(nums): 
            if num in dicti: 
                return [dicti[num], i]
            else: 
                dicti[target-num] = i
        
        return []